src/fides/api/main.py

```diff
@@ -296,6 +298,7 @@ async def setup_server() -> None:
         )
     )
 
+    # TODO: move this higher up
```

Comment:
Note to self: since we do the logging last, all the logging config isn't used for the rest of server startup... probably should be a separate ticket